runtime/ftplugin/man.vim

```diff
@@ -41,7 +37,6 @@ endif
 
 if exists(":Man") != 2
   com -nargs=+ Man call s:GetPage(<f-args>)
-  nmap <Leader>K :call <SID>PreGetPage(0)<CR>
```

Comment:
instead of mapping `<leader>K`, the default `K` behavior should call `:Man` (not done yet).